tutoroars/templates/oars/jobs/init/oars_init_schemas_tables_users.sh

```diff
@@ -109,14 +99,3 @@ GRANT INSERT, SELECT ON {{ OARS_COURSEGRAPH_DATABASE }}.* TO '{{ OARS_CLICKHOUSE
 GRANT SELECT ON {{ OARS_COURSEGRAPH_DATABASE }}.* TO '{{ OARS_CLICKHOUSE_REPORT_USER }}';
 
 EOF
-
-echo "Replacing demo coursegrah data ..."
-clickhouse client --user "{{ CLICKHOUSE_ADMIN_USER }}" --password="{{ CLICKHOUSE_ADMIN_PASSWORD }}" --host "{{ CLICKHOUSE_HOST }}" --port {{ CLICKHOUSE_PORT }} --allow_experimental_lightweight_delete=1 -q "DELETE FROM {{ OARS_COURSEGRAPH_DATABASE }}.{{ OARS_COURSEGRAPH_NODES_TABLE }} WHERE course_key = 'course-v1:edX+DemoX+Demo_Course'";
-
-clickhouse client --user "{{ CLICKHOUSE_ADMIN_USER }}" --password="{{ CLICKHOUSE_ADMIN_PASSWORD }}" --host "{{ CLICKHOUSE_HOST }}" --port {{ CLICKHOUSE_PORT }} --allow_experimental_lightweight_delete=1 -q "DELETE FROM {{ OARS_COURSEGRAPH_DATABASE }}.{{ OARS_COURSEGRAPH_NODES_TABLE }} WHERE course_key = 'course-v1:edX+DemoX+Demo_Course'";
-
-clickhouse client --user "{{ CLICKHOUSE_ADMIN_USER }}" --password="{{ CLICKHOUSE_ADMIN_PASSWORD }}" --host "{{ CLICKHOUSE_HOST }}" --port {{ CLICKHOUSE_PORT }} --query "INSERT INTO {{ OARS_COURSEGRAPH_DATABASE }}.{{ OARS_COURSEGRAPH_NODES_TABLE }} FORMAT CSV" \
-  < /app/oars/data/clickhouse/coursegraph_nodes.csv;
-
-clickhouse client --user "{{ CLICKHOUSE_ADMIN_USER }}" --password="{{ CLICKHOUSE_ADMIN_PASSWORD }}" --host "{{ CLICKHOUSE_HOST }}" --port {{ CLICKHOUSE_PORT }} --query "INSERT INTO {{ OARS_COURSEGRAPH_DATABASE }}.{{ OARS_COURSEGRAPH_RELATIONSHIPS_TABLE }} FORMAT CSV" \
-  < /app/oars/data/clickhouse/coursegraph_relationships.csv;
```

Comment:
@bmtcril I know this is useful, specially for development but for production this doesn't add value. Do you think this can be moved to a different command that can only be used in development?

Reply:
For sure, this can probably go away altogether.